app/mailers/admin_mailer.rb

```diff
@@ -40,10 +40,17 @@ def feedback(feedback_id)
     feedback = Feedback.find(feedback_id)
     @summary = feedback.summary
     @comment = feedback.comment
+    @username = if feedback.username.present?
```

Comment:
I think you can do `@username = feedback.username if feedback.username.present?` instead of writing it across three lines. (Same for email.)